sources/PseudoTerminal.m

```diff
@@ -6222,7 +6232,7 @@ - (void)safelySetSessionSize:(PTYSession*)aSession rows:(int)rows columns:(int)c
     }
     PtyLog(@"safelySetSessionSize");
     BOOL hasScrollbar = [self scrollbarShouldBeVisible];
-    if (windowType_ == WINDOW_TYPE_NORMAL || windowType_ == WINDOW_TYPE_NO_TITLE_BAR) {
+    if (windowType_ == WINDOW_TYPE_NORMAL || windowType_ == WINDOW_TYPE_NO_TITLE_BAR || windowType_ == WINDOW_TYPE_TOP) {
```

Comment:
I think you found a bug. This line should actually say:
````
if (![self anyFullScreen]) {
````
Since all window types besides full screen are now resizable.